indexing-hadoop/src/main/java/org/apache/druid/indexer/JobHelper.java

```diff
@@ -364,9 +365,9 @@ public static void ensurePaths(HadoopDruidIndexerConfig config)
   public static void writeJobIdToFile(String hadoopJobIdFileName, String hadoopJobId)
   {
     if (hadoopJobId != null && hadoopJobIdFileName != null) {
-      try {
+      try (final OutputStream out = java.nio.file.Files.newOutputStream(Paths.get(hadoopJobIdFileName))) {
```

Comment:
Could you replace the sole use of Guava's `Files` in this class with `java.nio.file.Files.copy()` and then avoid fully-qualified class name?

Reply:
Done.